framework/src/service/ServiceListeners.cpp

```diff
@@ -446,7 +446,11 @@ void ServiceListeners::GetMatchingServiceListeners(const ServiceEvent& evt,
       const LDAPExpr& ldapExpr = sse.GetLDAPExpr();
       if (ldapExpr.IsNull() || ldapExpr.Evaluate(props, false)) {
         set.insert(sse);
-      }
+      } else if (evt.GetType() == ServiceEvent::SERVICE_REGISTERED) {
+        const std::string msg = "Service listener matching LDAP filter could not be found using LDAP filter: " + sse.GetFilter();
+          coreCtx->logger->Log(
+            cppmicroservices::logservice::SeverityLevel::LOG_DEBUG, msg);
```

Comment:
thanks, this is much easier to understand. 
The service properties, in the `props` variable, should also be logged to show users the properties compared against when the LDAP filter failed to match.